DNN Platform/Library/Data/DataProvider.cs

```diff
@@ -3948,7 +3948,7 @@ public virtual int GetContentWorkflowStateUsageCount(int stateId)
             return ExecuteScalar<int>("GetContentWorkflowStateUsageCount", stateId);
         }
 
-        [Obsolete("Deprecated in Platform 7.4.0")]
+        [Obsolete("Deprecated in Platform 7.4.0. Scheduled removal in v11.0.0.")]
```

Comment:
I am not familiar enough with this, but we should provide some basic guidance on what to replace it with.